pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: pdftron_flutter
 description: A new flutter plugin project.
-version: 0.0.100
+version: 0.0.101
```

Comment:
missing api docs

Reply:
It would be part of `disabledElements` so it's just added to the `Buttons` in `constants.dart`